src/libsync/vfs/cfapi/vfs_cfapi.cpp

```diff
@@ -89,7 +89,7 @@ void VfsCfApi::stop()
 void VfsCfApi::unregisterFolder()
 {
     const auto localPath = QDir::toNativeSeparators(params().filesystemPath);
-    const auto result = cfapi::unegisterSyncRoot(localPath);
+    const auto result = cfapi::unregisterSyncRoot(localPath);
```

Comment:
too bad this typo fix wasn't in a separate commit, that clouds a bit the review

Reply:
@er-vin Indeed? Well, I thought it is just so small so it doesn't deserve all the hassle with new PR + CI + merge? Anyways, can we let it slide just this time?

Reply:
No no, I just meant separate commit in this PR. When there's more than one commit I review each one separately which helps quite a lot.

But yes don't bother changing it now.